### ISSUE TITLE
[a11y]: default placeholder font color doesn't satisfy WCAG minimum contrast requirements

### ISSUE THREAD

**Issue by k3ls33** [carbon-design-system/carbon#11426]
### Package

@carbon/react

### Browser

Firefox

### Operating System

MacOS

### Package version

v11.3.0

### Automated testing tool and ruleset

WCAG 2.1

### Description

<img width="451" alt="image" src="https://user-images.githubusercontent.com/104227548/168877157-d0ec5d3c-b9ca-4b40-86a1-a439c2f3b61b.png">

As seen above, the default color of placeholder text in Carbon themes has a 2.52 contrast ratio, which doesn't meet WCAG 2.1 minimum contrast requirements for text. According to WCAG, "The minimum contrast success criterion (1.4.3) applies to text in the page, including placeholder text and text that is shown when a pointer is hovering over an object or when an object has keyboard focus." ([source](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))

### WCAG 2.1 Violation

Success Criterion 1.4.3 Contrast (Minimum)

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-combobox--combobox

### Steps to reproduce

1. Navigate to the [ComboBox storybook example](https://react.carbondesignsystem.com/?path=/story/components-combobox--combobox) for carbon/react
2. Use a tool of your choice to check the contrast ratio between the placeholder text and background color (I used Firefox's built-in developer tools)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Thanks for opening this issue. We had a very similar [one here](19553), and our accessibility team was discussing it together with our design team. Here’s some more information based on what was discussed:

> text in the Search component is $text-placeholder.
> 
> From an accessibility standpoint, placeholder text isn’t required to meet color contrast standards. It’s intended to offer optional hints or examples to help users understand what to enter in the field. Because placeholder text disappears once the user begins typing, it should not replace a visible label or include essential information needed to type in a correct value in the input. This is currently working as intended and not failing a11y requirements, so we will be closing this issue.
https://github.com/carbon-design-system/carbon/issues/19553#:~:text=The%20color%20token,closing%20this%20issue.

Based on this, we are closing this issue. If you have any question, just let us know.
